website/docs/internals/json-format.mdx

```diff
@@ -60,9 +60,7 @@ Because state does not currently have any significant metadata not covered by th
   "terraform_version": "version.string"
 }
 ```
-
-The extra wrapping object here will allow for any extension we may need to add in future versions of this format.
-
+The extra wrapping object is intentional. This approach allows us to extend this format if required.
```

Comment:
I don't think we really need to mention this at all; it doesn't really matter _why_ this wrapping object is here, only that it is. :thinking:

Reply:
I'm glad you said that! I'm going to remove this then.